javascript/builtins/Array.json

```
@@ -5,6 +5,9 @@
         "__compat": {
           "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Array",
           "spec_url": "https://tc39.es/ecma262/multipage/indexed-collections.html#sec-array-objects",
+          "tags": [
+            "web-features:ecmascript-1"
```

The only question I have is whether we treat groups as a namespace in web-features and do this here:

```suggestion
            "web-features:group:ecmascript-1"
```

The arguments for and against are quite hypothetical now, but my hunch is that it's overall a good thing if the distinction between group and feature is clear, so that one doesn't have to wonder if a string like "web-features:grid" or "web-features:speech" refers to a group or concrete feature.